pages/design/rule-design.md

```diff
@@ -2,35 +2,69 @@
 title: Rule Design
 ---
 
-The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is suggested:
+The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting documents. To achieve the goals the following approach is based on [ACT Rules Format 1.0](https://www.w3.org/TR/act-rules-format/):
 
-1. **[Rule properties](#rule-properties)**: Define the test subject and its environment, as well as other meta data.
 
-2. **Description**: Brief description of what the rule does
+1. **[Rule properties](#rule-properties)**: Define the test subject and its environment. Includes identifier, name, type and description, as well as other meta data.
```

Comment:
Can we refer/ hyperlink to the perhaps a frontmatter section of the act rules format?